commands/new_function.go

```diff
@@ -27,20 +26,20 @@ func init() {
 	newFunctionCmd.Flags().StringVarP(&imagePrefix, "prefix", "p", "", "Set prefix for the function image")
 
 	newFunctionCmd.Flags().BoolVar(&list, "list", false, "List available languages")
-	newFunctionCmd.Flags().StringVarP(&appendFile, "append", "a", "", "Append to existing YAML file")
 
 	faasCmd.AddCommand(newFunctionCmd)
 }
 
 // newFunctionCmd displays newFunction information
 var newFunctionCmd = &cobra.Command{
-	Use:   "new FUNCTION_NAME --lang=FUNCTION_LANGUAGE [--gateway=http://domain:port] | --list | --append=STACK_FILE)",
+	Use:   "new FUNCTION_NAME --lang=FUNCTION_LANGUAGE [--gateway=http://domain:port] | --list | --yaml=YAML_FILE)",
 	Short: "Create a new template in the current folder with the name given as name",
-	Long: `The new command creates a new function based upon hello-world in the given
-language or type in --list for a list of languages available.`,
+	Long: `The new command creates a new function based upon hello-world in the given language
+or type in --list for a list of languages available.  YAML file can be specified by
+--yaml flag to generate new file otherwise it will append function to existing file`,
```

Comment:
I think this needs to more clearly express what will happen.  As I understand it if `stack.yml` exists then it will be appended to there unless `--yaml` is added and in that case if the file exists it will be appended to, otherwise a new file will result.